tests/functional/schema/test_custom_schema.py

```diff
@@ -202,18 +201,15 @@ def test__postgres__custom_schema_from_model_name(
         project.run_sql(_VALIDATION_SQL)
         run_dbt(["seed"])
         results = run_dbt(["run"])
-        assert len(results) == 3
+        # Model 2 is ephemeral, so it should not show up in the results list
+        assert len(results) == 2
```

Comment:
I hope this is the right place to test support for ephemeral models with dots in their names, but let me know if there's a better option!